src/frontend-admin/src/components/App/App.module.css

```diff
@@ -34,34 +35,70 @@
   letter-spacing: -1.38px;
 }
 
+.navEllipse {
+  position: relative;
+  left: 0px;
+  background: #333;
+  width: 40px;
+  height: 30px;
+  border-radius: 50%;
+  z-index: 3;
+}
+
+.navRectangle {
+  background: #f4f4f4;
+  position: relative;
+  top: 15px;
+  left: 40px;
+  width: 50px;
+  height: 20px;
+  border-radius: 0px 2px 0px 0px;
+  z-index: 3;
+}
+
+.navTabRectangle {
+  background: #333;
+  position: relative;
+  left: -50px;
+  width: 20px;
+  height: 20px;
+  border-radius:  0 0px 0px -18px; 
+}
+
 .linkContainer {
   align-self: flex-end;
   margin-right: 60px;
-  background: #333;
   border-radius: 0 18px 18px 18px; 
   display: flex;
   align-content: center;
   height: 80px;
+  flex-direction: row-reverse;
```

Comment:
I usually raise an eyebrow when I see this particular direction - sometimes it makes more sense to restructure your html in react to be in the correct order rather than switching the ordering via css. That way if someone looks at the component code it'll more accurately match what they see on the screen.

Reply:
It feels messy to leave it reversed but it was the only way I could find to layer the tabs correctly :(